docs/source/markdown/podman-system-prune.1.md

```diff
@@ -20,9 +20,9 @@ Recursively remove all unused pods, containers, images, networks, and volume dat
 
 #### **--external**
 
-Removes all leftover container storage files from local storage that are not managed by podman. In normal circumstances no such data should exist, but in case of an unclean shutdown the podman database may be corrupted and cause his.
+Removes all leftover container storage files from local storage that are not managed by podman. In normal circumstances no such data exists, but in case of an unclean shutdown the podman database may be corrupted and cause his.
```

Comment:
```suggestion
Removes all leftover container storage files from local storage not managed by Podman. In normal circumstances, no such data exists, but in case of an unclean shutdown, the Podman database may be corrupted and cause this.
```